infra/terraform/terraform.go

```diff
@@ -260,8 +261,8 @@ var (
 )
 
 // installerCommand waits for the installer tarball to download, unpacks it and launches the installation
-const installerCommand = `while [ ! -f /home/centos/installer.tar.gz ]; do sleep 5; done; \
-tar -xvf /home/centos/installer.tar.gz -C /home/centos/installer; \
-/home/centos/installer/install`
-
-const installerLogPath = "/home/centos/installer/gravity.log"
+func installerCommand(config Config) string {
```

Comment:
Better to pass the bare minimum - e.g. just the user name required for substitution.